src/SearchButton.js

```diff
@@ -0,0 +1,5 @@
+import React from "react";
+function SearchButton() {
+  return <button className="btn btn-primary">Search</button>;
```

Comment:
Don't you think we could make this component more reusable? Like, receiving the description text via props and the action to be triggered when you click on it?